docs/roman/associations/skycell_asn.rst

```diff
@@ -92,16 +69,16 @@ contribute data to. The association files will be json files with names based
 
 .. code-block:: text
 
-	r00991_<release product name>_v<visit_id>_<skycell name>_<filter>_coadd_asn.json
+	r00991_<release product name>_v<visit_id>_<skycell name>_<filter>_asn.json
 
 or for the selections above
 
 .. code-block:: text
 
```

Comment:
Can we update this file name example now to be one of the new style names?  Just replace r257dp63x98y83 with your favorite skycell name.

Reply:
270p65x48y69